code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -121,7 +121,7 @@
 	desc = "A kit with everything you need to hack into and disrupt the Station, AI, its cyborgs and the Security team. HACK THE PLANET!"
 	items = list(
 		/obj/item/melee/energy/sword/saber/blue, // 40TC
-		/obj/item/card/emag, // 30TC
+		/obj/item/autosurgeon/organ/syndicate/hackerman_deck, // 30TC
```

Comment:
this auto surgeon can't be refilled right?

Reply:
One use.
`/obj/item/autosurgeon/organ/syndicate/hackerman_deck
	desc = "A single use autosurgeon that contains a Binyat wireless hacking system. A screwdriver can be used to remove it, but implants can't be placed back in."
	uses = 1
	starting_organ = /obj/item/organ/internal/cyberimp/brain/hackerman_deck`